src/screens/Profile/Profile.js

```diff
@@ -779,11 +793,21 @@ class Profile extends React.Component<Props, State> {
               <SubHeading>SYSTEM</SubHeading>
             </ListSeparator>
 
+            <ProfileSettingsItem
+              key="joinBeta"
+              label={userJoinedBeta ? 'Leave Beta Testing' : 'Join Beta Testing'}
+              onPress={() => userJoinedBeta
+                ? setUserJoinedBeta(false)
+                : this.setState({ showJoinBetaModal: true })
+              }
+            />
```

Comment:
What do you think about having two different branches instead of checking the same condition over and over? i.e., something like:

```jsx
{userJoinedBeta ? (
  <ProfileSettingsItem
    key="joinBeta"
    label="Leave Beta Testing"
    onPress={() => setUserJoinedBeta(false)}
   />
) : (
  <ProfileSettingsItem
    key="joinBeta"
    label="Join Beta Testing"
    onPress={() => this.setState({ showJoinBetaModal: true })}
   />
)}
```

Even tho it results on more lines, I think it looks cleaner and more straightforward.

Reply:
I would go with this approach:
```
{userJoinedBeta ? (
  <ProfileSettingsItem
    key="joinBeta"
    label="Leave Beta Testing"
    onPress={() => setUserJoinedBeta(false)}
   />
)}
{!userJoinedBeta ? (
  <ProfileSettingsItem
    key="joinBeta"
    label="Join Beta Testing"
    onPress={() => this.setState({ showJoinBetaModal: true })}
   />
)}
```
we should try to avoid `else` part in render methods as it creates complexity

Reply:
Thanks 👍